File: Programmers/Lv.1/나머지가 1이 되는 수 찾기.java
/**
 * fileName : 나머지가 1이 되는 수 찾기.java
 * author :  한찬영
 * date : 2023-03-07
 * description :
 * ===========================================================
 * DATE                 AUTHOR                NOTE
 * -----------------------------------------------------------
 * 2023-03-07            한찬영             최초 생성
 */

class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for(int i=2;i<1000000;i++){
            if(n%i==1){
                answer = i;
                break;
            }
            else{
                continue;
            }
            
        }
        
        return answer;
    }
}

File: Programmers/Lv.1/나머지가 1이 되는 수 찾기.py
/**
 * fileName : 나머지가 1이 되는 수 찾기.py
 * author :  한찬영
 * date : 2023-04-03
 * description :
 * ===========================================================
 * DATE                 AUTHOR                NOTE
 * -----------------------------------------------------------
 * 2023-04-03            한찬영             최초 생성
 */

def solution(n):
    answer = 0
    
    for i in range(2,1000000):
        if(n%i==1):
            answer = i
            break
        
    return answer
